wp-content/plugins/civicrm/civicrm/ext/afform/core/ang/af/afForm.component.js

```
@@ -334,52 +334,72 @@
-    
-        // Check if the start date is in the past or today
-        if (startDate <= today) {
-          errorMessage += `Collections cannot start (${startDateValue || institutionStartDateValue}) today or in the past.\n`;
-          isValid = false;
-        }
-        // Check if the end date is in the past, today
-        if (endDate <= today) {
-          errorMessage += `Collections cannot end (${endDateValue || institutionEndDateValue}) today or in the past.\n`;
-          isValid = false;
+        if (ctrl.getFormMeta().name === 'afformInstitutionGoonjActivitiesIntent') {
+          var dateField = $element.find("af-field[name='Institution_Goonj_Activities.Start_Date'] .crm-form-date-wrapper input.crm-form-date").val();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'dateField'. Consider to delete it or rename it.

'dateField' is defined here:


(lint/suspicious/noRedeclare)